src/components/Map/components/ZoneInfos.tsx

```diff
@@ -40,7 +42,13 @@ const ZoneInfos = ({ map, draw }: { map: Map; draw: MapboxDraw }) => {
   useEffect(() => {
     setSummary(undefined);
     if (bounds) {
-      heatNetworkService.summary(bounds).then(setSummary);
+      zoneIndex.current += 1;
+      const currentZoneIndex = zoneIndex.current;
+      heatNetworkService.summary(bounds).then((result) => {
+        if (currentZoneIndex === zoneIndex.current) {
```

Comment:
What is the objective? 
If it is for canceling the promise, maybe we prefer use an `AbortController` ?

Reply:
yes, but an abortcontroller is more complexe to implement => https://trello.com/c/7nqNCEOl/75-create-an-abortcontroller-to-abort-useless-stats-calculation